src/gui/MainPane/TopBar.qml

```diff
@@ -62,7 +62,7 @@ Rectangle {
             label.color: theme.mainPane.topBar.nameVersionLabel
             toolTip.text: qsTr("Open project repository")
```

Comment:
The tooltip text here should indicate "Double click to open project repository", else the user might see the button highlighted on hover but think it doesn't work from a single click (usual behavior for buttons).